### PR TITLE
Code review section

### PR DESCRIPTION
**Pull request by abdulrahmanshr75**
- Added The first install.
- Added the Linters
- Added Readme File
- Fixed Some Errors in html and css Files

### CODE REVIEW

File: babel/index.babel [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (abdulrahmanshr75) [inline babel/index.babel:27]: In order to keep your repo professional and clean, you should consider removing all unnecessary files. Please check [this article](https://coderwall.com/p/g16jpq/keep-your-git-directory-clean-with-git-clean-and-git-trash) to read more about git trash and git clean

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments. 
+"use strict";
```
Comment (abdulrahmanshr75) [inline js/index.js:28]: Please remove the unnecessary Files because it is not a good coding practice to leave the unused files.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+          </li>
+          <li>
+            Server-side JavaScript extends the core language by supplying
+            objects relevant to running JavaScript on a server. For example,
+            server-side extensions allow an application to communicate with a
+            database, provide continuity of information from one invocation to
+            another of the application, or perform file manipulations on a
+            server.
+          </li>
+        </ul>
```
Comment (abdulrahmanshr75) [inline index.html:57]: Every Element should have a class in order to make the code cleaner so I suggest that you add classes to ``p``, ``ul``, and ``li``

**Review by abdulrahmanshr75 — COMMENTED** [review]
Good Job so far 👍
- No Linter Errors ✔️
But there is some changes that you need to deal with

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @abdulrahmanshr75, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All comments are very polite 🔝 
- [x] You even added some emojis, making it more human friendly 👽  

## Required Changes ♻️
- [ ] As mentioned in the [how to write a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md), you should make a suggestion instead of demanding changes. Maybe you could consider another approach, such as `In order to keep your repo professional and clean, you should consider removing all unnecessary files. Please check [this article](https://coderwall.com/p/g16jpq/keep-your-git-directory-clean-with-git-clean-and-git-trash) to read more about git trash and git clean`.
- [ ] Your last comment is a little confusing, you should consider re-writing it for clarification, maybe add a reference to justify your requirement instead of using "I think". 
- [ ] Keep in mind that comments use the markdown files notation. If you want to add a piece of code, remember to wrap it in ``. 

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulrahmanshr75 — COMMENTED** [review]
- Changed the Approach
- Clarified the comment
- wrapped the code elements in ``

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi STUDENT,  

Your project is complete!  Congratulations!  🎉💯🌟 




Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**